src/components/checkout-item/checkout-item.component.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { clearItemFromCart, addItem, removeItem } from '../../redux/cart/cart.actions';
+import './checkout-item.styles.scss';
+
+const CheckoutItem = ({ cartItem, clearItem, addItem, removeItem }) => {
+  const { name, imageUrl, price, quantity } = cartItem;
+  return (
+    <div className="checkout-item">
+      <div className="image-container">
+        <img src={imageUrl} alt="item" />
+      </div>
+      <span className="name">{name}</span>
+      <span className="quantity">
+        <div className="arrow" onClick={() => removeItem(cartItem)}>
+          &#10094;{' '}
+        </div>
+        <span className="value"> {quantity}</span>
+        <div className="arrow" onClick={() => addItem(cartItem)}>
+          {' '}
```

Comment:
Are `{' '}` necessary ? These must be taken care of, there are other instances of it, please take care of them.